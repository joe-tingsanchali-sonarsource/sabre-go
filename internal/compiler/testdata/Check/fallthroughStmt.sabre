package main

func main() {
    fallthrough
}