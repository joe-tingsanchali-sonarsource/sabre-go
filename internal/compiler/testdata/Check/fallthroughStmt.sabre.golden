>> 	    fallthrough
>> 	    ^^^^^^^^^^^ 
Error[internal/compiler/testdata/Check/fallthroughStmt.sabre:4:5]: fallthrough statement not within switch
